tfjs-layers/src/engine/container.ts

```diff
@@ -594,14 +594,24 @@ export abstract class Container extends Layer {
   loadWeights(weights: NamedTensorMap, strict = true) {
     const nameToWeight: {[name: string]: LayerVariable} = {};
     let totalWeightsCount = 0;
+    // get weights key from tensor map in order to check if it is from keras v3.
+    // e.g. dense/0
+    const key = Object.keys(weights)[0].split('/');
+    const isKerasV3 = this.isKerasKeras(key[key.length - 1]);
```

Comment:
What about change it to `const isKerasFormat = this.isKerasFormat` or maybe `isKerasKerasFormat`? Sorry for swinging about the name.